ts/Dashboards/Layout/Cell.ts

```diff
@@ -590,6 +590,16 @@ class Cell extends GUIElement {
         }
     }
 
+    public setLoadingState(enabled: boolean = true): void {
```

Comment:
- method description is missing
- IMO it should be an internal method ( use `@internal` to don't list it in the API)
- it toggles the class name so it should be called `toggleLoadingState`

Reply:
> it toggles the class name so it should be called toggleLoadingState

It toggles but always forces value (in default `true`). Should it be called `toggleLoadingState` anyway?

Reply:
hmm, in that case I guess you can leave it as it is 😉